unearthed-arcana/2017/20170206.xml

```diff
@@ -303,7 +303,7 @@
           <p class="indent">As an action, you can gain a base AC of 13 + your Constitution modifier if you aren’t wearing armor, and your skin assumes a stony appearance. This effect lasts until you end it as a bonus action, you are incapacitated, or you don armor other than a shield.</p>
         </description>
         <sheet>
-          <description>Your hit point maximum increases by 1, and it increases by 1 again whenever you gain a level in this class. As an action, gain a base AC of 13 +{{constitution:modifier}}. This effect lasts until you end it as a bonus action, you are incapacitated, or you don armor other than a shield.</description>
+          <description>Your hit point maximum increases by 1, and it increases by 1 again whenever you gain a level in this class. As an action, gain a base AC of 13 + {{constitution:modifier}}. This effect lasts until you end it as a bonus action, you are incapacitated, or you don armor other than a shield.</description>
```

Comment:
In the sheet description, you can also remove the `Your hit point maximum increases by 1, and it increases by 1 again whenever you gain a level in this class.`

And change you 13+ con to the calculated value of the AC `{{ac:stones durability}}`